src/renderers/BaseTimedMediaRenderer.js

```diff
@@ -146,30 +146,16 @@ export default class BaseTimedMediaRenderer extends BaseRenderer {
             (isLooping && this._accumulatedMediaTime >= duration)
         ) {
             // Some players fallback to first frame--force showing ending frame.
-            this._playoutEngine.setCurrentTime(this._rendererId, this._inTime + duration - 0.1);
+            this._playoutEngine.setCurrentTime(this._rendererId, duration-0.1);
```

Comment:
it might be safest to do the revert properly, but I suspect this change should be in regardless